jobs/openshift/cluster/integreatly/test/Jenkinsfile

```diff
@@ -0,0 +1,124 @@
+def testOptions = [:]
+
+def verifyParameters(testOptions) {
+  if (!testOptions.gitUrl || !testOptions.gitBranch) {
+    def userInput = input message: 'Integreatly Test Options', parameters: [
+      string(defaultValue: (testOptions.gitUrl ?: 'https://github.com/integr8ly/installation.git'), description: 'Integreatly installer Git URL', name: 'installerGitUrl'),
+      string(defaultValue: (testOptions.gitBranch ?: 'master'), description: 'Integreatly Git Branch', name: 'installerGitBranch'),
+    ]
+
+    testOptions.gitUrl = userInput.installerGitUrl
+    testOptions.gitBranch = userInput.installerGitBranch
+    testOptions.clusterName = userInput.installerGitBranch
+    testOptions.openshiftMasterUrl = "https://${userInput.installerGitBranch}.${testOptions.domainName}"
+    verifyParameters(testOptions)
+  }
+}
+
+pipeline {
+  agent {
+    node {
+      label 'cirhos_rhel7'
+    }
+  }
+
+  parameters {
+    string(defaultValue: 'https://github.com/integr8ly/installation.git', description: '[REQUIRED] Integreatly installer Git URL', name: 'installerGitUrl')
+    string(defaultValue: 'master', description: '[REQUIRED] The name of the git branch to be used for installing Integreatly', name: 'installerGitBranch')
+    booleanParam(defaultValue: false, description: '[OPTIONAL][Test] Dry run only, only prints what it would do!', name: 'dryRun')
+  }
```

Comment:
@JameelB Do you know if these are required here if we already have them in the job definition above?

Reply:
Can look into removing this duplication after.